kitchen.dokken.yml

```diff
@@ -1,7 +1,7 @@
 driver:
   name: dokken
   privileged: true # because Docker and SystemD/Upstart
-  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
```

Comment:
The goal here was to default to testing the current release, but fallback if we wanted to. If you wan to test Chef 13 you should be specifying the version

Reply:
Hello Tim. I didn't realise the main goal of this before i edited it... And I should ! Nice feature. 

I'll correct my work very soon. Thanks for the quick review.